src/binary/component.rs

```diff
@@ -17,10 +17,25 @@ pub enum ComponentType {
 #[cfg(any(feature = "backend-kvm"))]
 pub const PT_ENARX_SALLYPORT: u32 = PT_LOOS + 0x34a0001;
 
-/// The enarx code program header type
-#[cfg(any(feature = "backend-kvm"))]
+/// The enarx init program header type
 pub const PT_ENARX_CODE: u32 = PT_LOOS + 0x34a0003;
 
+/// This segment contains TCS pages.
+#[cfg(feature = "backend-sgx")]
+pub const PF_ENARX_SGX_TCS: u32 = 1 << 20;
+
+/// This segment contains unmeasured pages.
+#[cfg(feature = "backend-sgx")]
+pub const PF_ENARX_SGX_UNMEASURED: u32 = 1 << 21;
```

Comment:
Where is this used in the linker script?

Reply:
It isn't (yet?). But I want it well defined in case we choose to use this strategy in the shim in the future. That way it won't require a new server side. Only client side changes will be required.